books.js

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable no-unused-vars */
+import BookClass from './modules/bookclass.js';
```

Comment:
- [ ] Let's make sure that we do not disable the linter check because it makes us write better and clean code, and it is also required not to disable the linters check. Kindly look for the variable that was declared but not used and fix the error.

Reply:
Removed Linter Disable from code